src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -368,7 +368,7 @@ export class MeetingManager implements AudioVideoObserver {
           console.error('MeetingManager failed to get device permissions');
           this.deviceLabelTriggerStatus = DeviceLabelTriggerStatus.DENIED;
           this.publishDeviceLabelTriggerStatus();
-          throw new Error(error);
```

Comment:
We discussed this earlier and agreed that it's fine to make this change. The `error` here is the original `getUserMedia` errors, for example `NotReadableError`. These errors are of type `DOMException` which is inherited from `Error` prototype, so it's not a breaking change. 

We wrap these errors with a general `Error` type which is not helpful for builder because they are not able to check the name of the error (`error.name` is always "Error"). And this is why we return it directly here.

Reply:
Cool thanks for the context.